package.json

```diff
@@ -85,7 +86,15 @@
                 "azure-pipelines.configure": {
                     "type": "boolean",
                     "default": true,
-                    "description": "Enable 'Configure Pipeline' feature"
+                    "description": "Enable 'Configure Pipeline' feature",
+                    "order": 0
+                },
+                "azure-pipelines.1ESPipelineTemplatesSchemaFile": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Use 1ES Pipeline Template schema file",
+                    "markdownDescription": "If enabled, [1ES Pipeline Template schema file](https://aka.ms/1espt) will take precedence over custom schema file when user is signed in with Microsoft account and the schema is available in the ADO organization.",
+                    "scope": "machine-overridable"
```

Comment:
```suggestion
```
I don't think this is needed - this setting seems like it should sync seamlessly across machines (unlike the customSchemaFile setting, which uses local filepaths).

Reply:
Removed it